test/test.integration-all-services.js

```diff
@@ -605,6 +605,11 @@ describe('integration-all-services', function() {
       }, failIfError.bind(failIfError, done));
     });
 
+    it('getImageSceneText()', function(done) {
+      alchemy_vision.getImageSceneText({
+        url: 'http://itsopen.co.uk/site/wp-content/themes/its-open/assets/images/logo-its-open-social-media.png'
```

Comment:
Who owns that website? If it isn't us, then we should probably find somewhere else to host the image (I'm thinking we stick it in with the docs on the gh-pages branch)